src/components/Login/index.tsx

```diff
@@ -71,51 +74,52 @@ export const FormComp = (props: StyleProps & FormikProps<FormValues>) => {
   )
 }
 
-interface StyleProps {
-  classes: {
-    grid: string
-    form: string
-  }
-}
-
-const styles = createStyles({
-  form: {
-    display: "flex",
-    minHeight: "100vh",
-  },
-  grid: {
-    display: "flex",
-    justifyContent: "center",
-    alignContent: "center",
-    color: "red",
-  },
-})
-
-export const LoginForm = withStyles(styles)(FormComp)
+//export const LoginForm = withStyles(styles)(FormComp)
```

Comment:
@Alexandria ⛏ (nitpick) Make sure to remove commented out lines before pushing up your code.